app/controllers/heartbeat_controller.rb

```diff
@@ -2,11 +2,19 @@ class HeartbeatController < ApplicationController
   permit_only_with_key
 
   def healthcheck
-    render json: {
+    messagelabs_alive = messagelabs_alive?
+    sendgrid_alive = sendgrid_alive?
+    database_alive = ActiveRecord::Base.connection.active?
+    status = status = :bad_gateway unless [
```

Comment:
Arg. Typo. I've also just realised there's a nicer way to do this whole block using a hash with `.values.any?`.